.github/workflows/ci.yml

```diff
@@ -455,22 +475,10 @@ jobs:
         with:
           python-version: 3.8
 
-      # local synapse
-      - uses: michaelkaye/setup-matrix-synapse@main
-        with:
-          uploadLogs: true
-          httpPort: 8228
-          disableRateLimiting: true
-
-      # latest sliding sync proxy
-
-      - uses: addnab/docker-run-action@v3
-        with:
-          registry: gcr.io
-          image: "matrix-org/sliding-sync:v0.98.0"
-          docker_network: "host"
-          options: '-e "SYNCV3_SERVER=http://locahost:8228" -e "SYNCV3_SECRET=SUPER_CI_SECRET" -e "SYNCV3_BINDADDR=:8118" -e "SYNCV3_DB=user=postgres password=postgres dbname=syncv3 sslmode=disable host=postgres" -p 8118:8118'
-
       - name: Test
         run: |
-          cargo nextest run -p sliding-sync-integration-tests
+          RUST_LOG=hyper=trace \
+          HOMESERVER_URL=http://localhost:8008 \
+          HOMESERVER_DOMAIN=synapse \
+          SLIDING_SYNC_PROXY_URL=http://localhost:8118 \
```

Comment:
Instead of making these part of `run:`, they can be `env:` fields in YAML, right? Then we don't need the newline escaping.